### ISSUE TITLE
Merge django-badger into this project?

### ISSUE THREAD

**Issue by lmorchard** [mozilla/badges.mozilla.org#92]
This project was started as a minimal site wrapper using django-badger. And, django-badger was created as a separate django app in the hopes that other sites would use it.

Unfortunately, that has not happened. And, having django-badger as a separate project in a separate repo makes working on badges.m.o more difficult than I'd like. So, it might help to pull django-badger in and make it a part of the badges.m.o codebase.

This *may* be a terrible idea worth pondering before doing, of course.

**Comment by lmorchard** [comment]
Okay, so what all the above commit thrashery signifies is this:

* I first tried merging by just copying files over and then moving things around. This was a mistake because:
    1. It broke a bunch of things because I renamed directories.
    2. I lost all the commit history for the files moved.
* Then, I tried `git filter-branch` a few times until I got a clean transplant of the `badges` subdirectory with history and no conflicts with the broader project history.
* Then, I got tests passing.
